frontend/src/components/common/SendEmailNotification/SendEmailNotification.tsx

```diff
@@ -0,0 +1,26 @@
+import emailjs from 'emailjs-com';
+export const sendEmailNotification = (
+    to_email: string,
+    to_name: string,
+    subject: string,
+    message: string
+): void => {
+    const templateParams = {
+        to_email,
+        to_name,
+        subject,
+        message,
+    };
+
+    import('emailjs-com')
+        .then(emailjs => {
+            emailjs
+                .send('service_rkmqc9w', 'template_h8ylqcg', templateParams, 'yo_kKRN2eG451iE_u')
```

Comment:
Can you add the service ID as an environment variable in the `.env` file?

Reply:
Here is the sample email screenshot:
![image](https://github.com/mursal-ashraf/bankan/assets/100513480/7b7f0cd1-6c98-4ef0-b72e-abb9623a7436)

Reply:
I created an account named: kanban.official@gmail.com, the password is kanbanboard. Seems like a professional team XD